src/pages/guidelines/layout/code.mdx

```diff
@@ -0,0 +1,1174 @@
+---
+label: The grid is the framework for all the visual elements and typography at IBM. It’s fundamental to everything we design.
+title: Layout
+tabs: ['Overview', 'Code']
+---
+
+<AnchorLinks>
+
+<AnchorLink>Under the hood</AnchorLink>
+<AnchorLink>Getting started</AnchorLink>
+<AnchorLink>How it works</AnchorLink>
+<AnchorLink>Grid modes</AnchorLink>
+<AnchorLink>Incorporating the Shell</AnchorLink>
+<AnchorLink>Alignment</AnchorLink>
+<AnchorLink>Techniques</AnchorLink>
+
+</AnchorLinks>
+
+## Under the hood
+
+The IBM 2x Grid system is a fully responsive, robust, 16 column layout system with five default breakpoints and dozens of predefined classes.
+
+Our grid is built with Flexbox and closely mirrors Bootstrap's grid in that it uses containers, rows and columns to layout and align content. Below is an in-depth look at how it all comes together.
+
+If you’re new to Flexbox, this [CSS Tricks Flexbox guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) will provide more detail around everything from basic terminology to code.
+
+## Getting started
+
+### With Carbon-components
+
+We provide multiple ways for you to use the grid for the Carbon Design System. By default, the grid code is included when you use `carbon-components`. To get started, you'll need to install this package using `npm`:
+
+```bash
+npm install carbon-components --save
+```
+
+<br />
+
+Or, if you prefer Yarn:
+
+```bash
+yarn add carbon-components
+```
+
+<br />
+
+You can include the grid in one of two ways:
+
+```scss
+// If you're including all the styles for Carbon
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+// If you're wanting to import the grid
+@import 'carbon-components/scss/globals/grid/grid';
+```
+
+<br />
+
+To use 16 column variant of the grid, you need to enable the `grid-columns-16` feature flag before including any imports in your Sass files. For example:
+
+```scss
+$feature-flags: (
+  grid-columns-16: true,
+);
+
+// If you're including all the styles for Carbon
+@import 'carbon-components/scss/globals/scss/styles.scss';
+
+// If you want to import the grid
+@import 'carbon-components/scss/globals/grid/grid';
+```
+
+<InlineNotification>
+
+**Note:** By default, Carbon loads with 12 columns. This will change in v11 and the grid will default to 16 columns. Until then we ask you to enable the feature flag.
```

Comment:
```suggestion
**Note:** By default, Carbon loads with 12 columns. In a later release this will change and the grid will default to 16 columns. Until then we ask you to enable the feature flag.
```